web/profiles/custom/yalesites_profile/config/sync/paragraphs.paragraphs_type.media_grid_item.yml

```diff
@@ -4,7 +4,7 @@ status: true
 dependencies: {  }
 id: media_grid_item
 label: 'Image Grid Item'
-icon_uuid: null
-icon_default: null
+icon_uuid: 8c3aae68-7b35-481f-b559-b15512b1e34f
+icon_default: 'data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDI3LjAuMCwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCA1NzYgNTEyIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA1NzYgNTEyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+CjxzdHlsZSB0eXBlPSJ0ZXh0L2NzcyI+Cgkuc3Qwe2Rpc3BsYXk6bm9uZTt9Cgkuc3Qxe2ZpbGw6bm9uZTt9Cjwvc3R5bGU+CjxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0wLDEyOGMwLTM1LjMsMjguNy02NCw2NC02NGgyNTZjMzUuMywwLDY0LDI4LjcsNjQsNjR2MjU2YzAsMzUuMy0yOC43LDY0LTY0LDY0SDY0Yy0zNS4zLDAtNjQtMjguNy02NC02NFYxMjh6CgkgTTU1OS4xLDk5LjhjMTAuNCw1LjYsMTYuOSwxNi40LDE2LjksMjguMnYyNTZjMCwxMS44LTYuNSwyMi42LTE2LjksMjguMnMtMjMsNS0zMi45LTEuNmwtOTYtNjRsLTE0LjItOS41VjMyMFYxOTJ2LTE3LjFsMTQuMi05LjUKCWw5Ni02NEM1MzYsOTQuOSw1NDguNiw5NC4yLDU1OS4xLDk5LjhMNTU5LjEsOTkuOHoiLz4KPHJlY3QgeD0iMTEiIHk9Ii0yMSIgY2xhc3M9InN0MSIgd2lkdGg9IjU3NCIgaGVpZ2h0PSI1NzQiLz4KPHBhdGggZD0iTTI0Mi4yLDE5MS44TDE3OC4zLDQ4LjVjLTIuMS0zLTUuNS00LjktOS4yLTQuOXMtNy4xLDEuOC05LjEsNC45TDExNywxMjZsLTE1LjMtMjNjLTIuMS0yLjktNS4zLTQuNS04LjgtNC41CglzLTYuOCwxLjctOC45LDQuNWwtNDkuMSw4OC44Ii8+CjxjaXJjbGUgY3g9IjY3IiBjeT0iMzcuOCIgcj0iMzYuOCIvPgo8cmVjdCB4PSIyNzkuMiIgeT0iMC41IiB3aWR0aD0iMTcuNiIgaGVpZ2h0PSI1MTEuMSIvPgo8cmVjdCB5PSIyNDcuMiIgd2lkdGg9IjU3NiIgaGVpZ2h0PSIxNy42Ii8+CjxwYXRoIGQ9Ik01NTIuNywxOTEuOEw0ODguOCw0OC41Yy0yLjEtMy01LjUtNC45LTkuMi00LjlzLTcuMSwxLjgtOS4xLDQuOWwtNDMsNzcuNWwtMTUuMy0yM2MtMi4xLTIuOS01LjMtNC41LTguOC00LjUKCXMtNi44LDEuNy04LjksNC41bC00OS4xLDg4LjgiLz4KPGNpcmNsZSBjeD0iMzc3LjUiIGN5PSIzNy44IiByPSIzNi44Ii8+CjxwYXRoIGQ9Ik0yNDIuMiw1MDEuMWwtNjMuOS0xNDMuM2MtMi4xLTMtNS41LTQuOS05LjItNC45cy03LjEsMS44LTkuMSw0LjlsLTQzLDc3LjVsLTE1LjMtMjNjLTIuMS0yLjktNS4zLTQuNS04LjgtNC41CglzLTYuOCwxLjctOC45LDQuNWwtNDkuMSw4OC44Ii8+CjxlbGxpcHNlIHRyYW5zZm9ybT0ibWF0cml4KDAuMDMzMyAtMC45OTk0IDAuOTk5NCAwLjAzMzMgLTI4Mi4xMzc0IDQwMi41MTIxKSIgY3g9IjY3IiBjeT0iMzQ3LjEiIHJ4PSIzNi44IiByeT0iMzYuOCIvPgo8cGF0aCBkPSJNNTUyLjcsNTAxLjFsLTYzLjktMTQzLjNjLTIuMS0zLTUuNS00LjktOS4yLTQuOXMtNy4xLDEuOC05LjEsNC45bC00Myw3Ny41bC0xNS4zLTIzYy0yLjEtMi45LTUuMy00LjUtOC44LTQuNQoJcy02LjgsMS43LTguOSw0LjVsLTQ5LjEsODguOCIvPgo8Y2lyY2xlIGN4PSIzNzcuNSIgY3k9IjM0Ny4xIiByPSIzNi44Ii8+Cjwvc3ZnPgo='
```

Comment:
@ShaneMcPartlinYale This one can be reverted back to the `null` value. A media_grid_item will never appear in the paragraph picker interface, so it does not need an icon. This "item" refers to a single image so the icon added in this PR kinda misrepresents its purpose as the icon shows four images.

Reply:
My mistake, I must have gotten lost in the mix for this one. Will address promptly.